cmd/broker.go

```diff
@@ -46,94 +48,91 @@ var brokerCatalogCmd = &cobra.Command{
 var brokerBootstrapCmd = &cobra.Command{
 	Use:   "bootstrap",
 	Short: "Bootstrap an Automation Broker instance",
-	Long:  `Fetch list of service bundles in Automation Broker catalog`,
+	Long:  `Refrsh list of bootstrapped service bundles in Automation Broker catalog`,
 	Run: func(cmd *cobra.Command, args []string) {
-		listBroker()
+		bootstrapBroker()
 	},
 }
 
 func init() {
+	brokerCmd.PersistentFlags().StringVarP(&brokerName, "name", "n", "openshift-automation-service-broker", "Name of Automation Broker instance")
 	rootCmd.AddCommand(brokerCmd)
-	// Broker List Flags
-	brokerCmd.Flags().StringVar(&brokerName, "name", "automation-broker", "Name of Automation Broker instance")
-
-	//Broker Bootstrap Flags
-	brokerBootstrap.Flags().StringVar(&removeName, "name", "", "Name of registry adapter to remove")
-	registryRemoveCmd.MarkFlagRequired("name")
-
-	registryCmd.AddCommand(registryAddCmd)
-	registryCmd.AddCommand(registryListCmd)
-	registryCmd.AddCommand(registryRemoveCmd)
-}
 
-func updateCachedRegistries(regList []Registry) error {
-	viper.Set("Registries", regList)
-	viper.WriteConfig()
-	return nil
+	brokerCmd.AddCommand(brokerCatalogCmd)
+	brokerCmd.AddCommand(brokerBootstrapCmd)
 }
 
-func addRegistry() {
-	var regList []Registry
-	err := viper.UnmarshalKey("Registries", &regList)
+func listBrokerCatalog() {
+	log.Debugf("func::listBrokerCatalog()")
+	var brokerRoute = ""
+	kube, err := clients.Kubernetes()
 	if err != nil {
-		fmt.Println("Error unmarshalling config: ", err)
+		log.Errorf("Failed to connect to cluster: %v", err)
 		return
 	}
-	for _, reg := range regList {
-		if reg.Config.Name == registryConfig.Config.Name {
-			fmt.Printf("Error adding registry [%v], found registry with conflicting name [%v]\n", registryConfig.Config.Name, reg.Config.Name)
-			return
-		}
+	ocp, err := clients.Openshift()
+	if err != nil {
+		log.Errorf("Failed to connect to cluster: %v", err)
+		return
 	}
-	regList = append(regList, registryConfig)
-	updateCachedRegistries(regList)
-	return
-}
 
-func printRegistries(regList []Registry) {
-	colName := &util.TableColumn{Header: "NAME"}
-	colType := &util.TableColumn{Header: "TYPE"}
-	colOrg := &util.TableColumn{Header: "ORG"}
-	colURL := &util.TableColumn{Header: "URL"}
-
-	for _, r := range regList {
-		colName.Data = append(colName.Data, r.Config.Name)
-		colType.Data = append(colType.Data, r.Config.Type)
-		colOrg.Data = append(colOrg.Data, r.Config.Org)
-		colURL.Data = append(colURL.Data, r.Config.URL)
+	// Attempt to get route of Automation Broker
+	rc, err := ocp.Route().Routes(brokerName).List(metav1.ListOptions{})
+	if err != nil {
+		log.Errorf("Failed to list routes in namespace %v: %v", brokerName, err)
 	}
 
-	tableToPrint := []*util.TableColumn{colName, colType, colOrg, colURL}
-	util.PrintTable(tableToPrint)
-}
+	for _, route := range rc.Items {
+		if route.Spec.To.Name == brokerName {
+			brokerRoute = route.Spec.Host
```

Comment:
feels like we should break right after this. No need to continue looping if it is found.